js/bundle_test.go

```diff
@@ -58,11 +62,11 @@ func TestNewBundle(t *testing.T) {
 	})
 	t.Run("Invalid", func(t *testing.T) {
 		_, err := getSimpleBundle("/script.js", "\x00")
-		assert.Contains(t, err.Error(), "SyntaxError: /script.js: Unexpected character '\x00' (1:0)\n> 1 | \x00\n")
+		assert.Contains(t, err.Error(), "SyntaxError: file:///script.js: Unexpected character '\x00' (1:0)\n> 1 | \x00\n")
```

Comment:
This seems strange and slightly wrong to me - why should the user see the `file` schema in the error messages if they didn't explicitly specify the schema themselves?

Reply:
Maybe we should print both the original thing that the user wrote and what we resolved it to as in [loader.Load](https://github.com/loadimpact/k6/pull/1059/files#diff-b38e43f0fb016af3e8cecd2d12d779ecR128)
I find the resolved final url to be even more useful as otherwise you don't actually know what we resolved whatever you provided us with

Reply:
>  I find the resolved final url to be even more useful as otherwise you don't actually know what we resolved whatever you provided us with

I don't understand what you mean by this, sorry

Reply:
if you have two scripts which have `require("./scripts.js")` but are in different directories and our message only tells you `./script.js ... had an error  ... ` you will have trouble to find out which `./script.js` it is. This gets even worse if you have a `https` import which imports `./script.js` or the more likely `./utils.js `. 
But yeah I agree that having the actual name of what the user provided is a good idea.

Reply:
> if you have two scripts which have require("./scripts.js") but are in different directories and our message only tells you ./script.js ... had an error ... you will have trouble to find out which ./script.js it is. 

In my head, the clear way to solve this is that both the originating file and the problem should be part of the log message, something like this:
```
/dir1/script1:line1234: cannot open "whatever the user wanted to open"
/dir2/script2:line3456: cannot open "whatever the user wanted to open"
```

Reply:
oops, wrong resolve... in both cases, `whatever the user wanted to open` should be exactly the string the user specified though

Reply:
so... is this fixable?

Reply:
I would consider the new one an improvement on the previous error. And what you propose to be a separate PR.